pkg/action/install.go

```diff
@@ -584,14 +592,107 @@ func (c *Configuration) renderResources(ch *chart.Chart, values chartutil.Values
 		}
 	}
 
-	if pr != nil {
-		b, err = pr.Run(b)
+	return hs, b, notes, nil
+}
+
+// postProcessFiles sends files for post-processing.
+// Some post-processing may need all objects at once.
+// Therefore we concatenate all rendered files, post-process and split them back.
+// A temporary annotation is added to preserve template filename.
+func (c *Configuration) postProcessFiles(files map[string]string, pr postrender.PostRenderer) (map[string]string, error) {
+	if pr == nil {
+		return files, nil
+	}
+
+	annotationName := "helmPostProcessFileName"
```

Comment:
This should be a `const` and not defined in the function. It should probably also be prefaced with a `helm.sh/`